packages/ui/src/Popup/Accounts/NetworkSelector/index.tsx

```diff
@@ -43,13 +84,15 @@ export function NetworkSelector({ onSelect }: NetworkSelectorProps) {
   // Automatically switch to 'Mainnet' network if current network doesn't exist.
   // This is necessary to prevent errors and UI bugs, as sometimes networks have to be modified, or removed.
   useEffect(() => {
-    if (!networkLabels[currentNetwork]) onSelect(NetworkName.mainnet);
+    // TODO: add logic for custom
```

Comment:
I don't think we need to support custom labels, unless there is a particular need?